src/native/corehost/error_codes.h

```diff
@@ -51,6 +51,7 @@ enum StatusCode
     HostPropertyNotFound                = 0x800080a4,
     CoreHostIncompatibleConfig          = 0x800080a5,
     HostApiUnsupportedScenario          = 0x800080a6,
+    HostFeatureDisabled                 = 0x800080a7,
```

Comment:
Please add this to https://github.com/dotnet/runtime/blob/main/docs/design/features/host-error-codes.md as well.

Reply:
Should have been there in my commit